content/en/docs/Concepts/olm-architecture/dependency-resolution/_index.md

```diff
@@ -38,23 +38,52 @@ Under the hood, these properties and constraints are converted into a system of
 
 ## Declaring Properties
 
-Properties cannot be directly declared by an operator author. Instead, they are derived from the existing metadata provided in the CSV.
-
 All operators in a catalog will have these properties:
 
 - `olm.package` - includes the name of the package and the version of the operator
 - `olm.gvk` - one property is present for each "provided" API from the ClusterServiceVersion
 
-A future release of OLM will support explicit property creation.
+Additional properties can also be directly declared by an operator author by including a `properties.yaml` file in the `metadata` directory of the operator bundle. See [Arbitrary Properties](#arbitrary-properties) for more information.
+
+```yaml
+properties:
+- type: olm.kubeversion
+  value:
+    version: "1.16.0"
+```
+
+## Arbitrary Properties
```

Comment:
Since you are now introducing the new `properties.yaml`, do you want to also include the versions of OLM that support this at the bottom of the document similarly to the `dependency.yaml` comment?